ivy/array/sorting.py

```diff
@@ -17,7 +17,24 @@ def argsort(
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
         return ivy.argsort(self._data, axis, descending, stable, out=out)
+        """
+        ivy.Array instance method variant of ivy.argsort. This method simply wraps the
+        function, and so the docstring for ivy.argsort also applies to this method
+        with minimal changes.
         
```

Comment:
You need to add a section for `Parameters` and `Returns` referring to the [Docstrings](https://lets-unify.ai/ivy/deep_dive/12_docstrings.html) page in the docs.